packages/cli/src/tools/copyAndReplace.js

```diff
@@ -131,4 +131,23 @@ function copyBinaryFile(srcPath, destPath, cb) {
   }
 }
 
+/**
+ * extended directory expectation.
+ * npm installs the directory of a local node module as a link to the folder of origin location.
+ * such the folder should be allowed to copy and replace as a template
+ */
+function isDirectory(srcPath) {
+  let srcStat = fs.lstatSync(srcPath);
+
+  if (srcStat.isSymbolicLink()) {
+    try {
+      return fs.readDirSync(srcPath) || false;
```

Comment:
wouldn't it be better to use `fs.realpathSync`?

Reply:
@thymikee 
It could be. In such the case, it is required to make an additional API call to FileSystem to get the stat of the real path entry (dir or file).
The current approach doesn't require it.

Reply:
Also, there's no `readDirSync` but `readdirSync`, are you sure this code works as intended? I've updated it, but would be cool if you verified it works for you

Reply:
@thymikee 
Thanks a lot for fixing of my fault :)